### DISCUSSION TITLE
Animated GLB Character in Hubs ALWAYS Disappears :/

### DISCUSSION THREAD

**Discussion by carlagnyu** [mozilla/hubs#3722] [General Questions & Discussion] [answered]
Hi All, 

I have a custom animated .fbx character I created (not as an avatar but to be used as a character in my Spoke scene). I tried exporting it (100+ x's  :) from Blender. The animation always disappears as soon as my mouse comes in contact with it, in the Spoke editor and in the published hubs scene. I'm in Visual Studio Code now, seeing how I might be able to edit the .gltf to work, but I'm really at a loss on where to begin. I've got a working character linked from sketchfab (but not my own char!!!). When I download the source files from this one (tried .fbx and .glTF) and export from Blender, it also disappears on mouse contact in Spoke and published hubs scene?? Any help would be greatly appreciated. Thank you!!  (see files attached, my model in image 02 disappears, the sketchfab model in image 03 does NOT disappear)

Thank you!
CG

![Capture01](https://user-images.githubusercontent.com/55764072/104821868-d9b06500-580c-11eb-9aff-9194380f9f70.JPG)
![Capture02](https://user-images.githubusercontent.com/55764072/104821869-d9b06500-580c-11eb-9b58-ddc9c22eda1e.JPG)
![Capture03](https://user-images.githubusercontent.com/55764072/104821870-d9b06500-580c-11eb-8766-9a3ea999251d.JPG)

**Comment by pattersonbl2** [comment]
are you still having issues with this?

**Comment by carlagnyu** [comment, marked as answer]
Nope, I figured it out :) Thanks for checking in.

Best,
*Carla Gannis*
*Industry Professor*
Department of Technology, Culture & Society
Integrated Digital Media Program
Tandon School of Engineering
*New York University*
--
Pronouns: she, her, hers

**Reply by pattersonbl2** [reply]
I will mark this as answered. If you can remember the fix please add it below to help other community members with the same issue.

**Comment by roadamico** [comment]
Hello, was this fix ever posted? I am quite curious about it too! Thanks so much!
Lucia Binotti, at UNC-CH working on webXR, Hubs and Task-Based Language Learning :-)
